4.5 release page (#1138)
godotengine/godot-website: collections/_article/godot-4-5-making-dreams-accessible.md

---
title: Godot 4.5, making dreams accessible

> excerpt: From mind-blowing effects now made possible by the stencil buffer to accessibilty descriptions of your GUI elements that opens up the possibility for some with disabilities to play your game — we are proud to present to you Godot 4.5.

Reviewer: ```suggestion
excerpt: From mind-blowing effects now made possible by the stencil buffer to accessible descriptions of your GUI elements that opens up the possibility for people with disabilities to play your game — we are proud to present to you Godot 4.5.
```

categories: ["release"]
author: "Godot contributors"
image: /storage/blog/covers/godot-4-5-making-dreams-accessible.webp
date: 2025-09-01 15:45:00
redirect_to:
 - /releases/4.5/
---

> Look forward to plenty of quality of life improvements hidden within this release. Faster load speeds, reduced stutter, streamlined processes — spotting all the optimizations that have been applied in the background will take some time.

Reviewer 2: For me at least, the sentence ending with "...will take some time" just leaves the idea of things "taking time" as the last thought in my head, which is probably the opposite of what we want here 😅 If others don't have the same instinct then this can be disregarded, but for now I think it might be worth considering if there's a different way it could be phrased, that puts more of an emphasis on speed itself.

Author: That text is the same as the blog post for the last release. I should maybe reword it a little bit (even if 99% of the people will never read it).

> On top of that, long-awaited comfort features like embedded game windows and interactive in-game editing will feel more in line with other software on the market, making transitioning between them an even smoother experience. The editor overhaul alone takes up a big chunk of this release, after all.

Reviewer 2: What is the "editor overhaul" being referred to here? The "Editor" section certainly has some nice improvements, but nothing sticks out to me as quite an "overhaul".

Author: Again, this was the last release text I need to change. Sorry.

We invite you to take a look over at the [4.5 release page](/releases/4.5/).

Or grab 4.5 here directly and get started.

{% include articles/download_card.html version="4.5" release="stable" article=page %}

> **Standard build** includes support for GDScript and GDExtension.

Reviewer 2: ```suggestion
The **Standard build** includes support for GDScript and GDExtension.
```
I'm not sure if "Standard" here should be capitalized, if we make it no longer the first word in the sentence? Depends if it's considered a proper noun or just an adjective, I suppose.

> **.NET build** (marked as `mono`) includes support for C#, as well as GDScript and GDExtension.

Reviewer 2: ```suggestion
The **.NET build** (marked as `mono`) includes support for C#, as well as GDScript and GDExtension.
```
